ui/src/pages/Logs/Logs.tsx

```diff
@@ -175,9 +198,40 @@ const logsDisplayStyle = {
   overflow: 'hidden',
   fontFamily: 'monospace',
 };
+
+const colorArray: string[] = [
+  purple[100],
+  red[100],
+  blue[100],
+  teal[100],
+  green[100],
+  cyan[100],
+  orange[100],
+  yellow[100],
+  brown[100],
+  blueGrey[100],
+];
+// Keeps track of what color to assign to each unique container
```

Comment:
There are a lot of "global" variables added to this file which are now in the closure of both `Logs` and `Row`. It would be better to implement a solution that does not add so many variables to the top-level scope and instead passes down data via props.

Reply:
If we decide to put the Row component into its own file (called `LogsRow`) .... this solution would no longer work and ultimately you would need to figure out how to pass data down via props.